### ISSUE TITLE
when closing current file, shutdown the Bracey automatically

### ISSUE THREAD

**Issue by cybaol** [turbio/bracey.vim#38]
Could you implement that when closing current file, shutdown/stop the Bracey automatically

**Comment by DONofDESTROY** [comment]
@cybaol  If I am correct Bracey already has this feature by default

**Comment by cybaol** [comment]
you are correct, but when closing files, server is closed immediately , howerer the broswer is not closed
